gtest_parallel_tests.py

```diff
@@ -414,6 +414,28 @@ def test_times_worker():
 
 
 class TestFilterFormat(unittest.TestCase):
+  def test_log_file_names(self):
+    self.assertEqual(
+      'bin-Test_case-100.log',
+      gtest_parallel.Task._logname('', 'bin', 'Test.case', 100))
+
+    self.assertEqual(
+      '../a/b/bin-Test_case_2-1.log',
```

Comment:
Should these be using `os.path.join(['..','a','b','bin...log'])` to get platform-dependent paths? Same for the input names? I'm guessing these tests might fail on Windows now which would use `..\a\b\bin-Test..log`?

Reply:
Oh, sorry. My fault. Now it definitely works on Windows.